R/sits_config.R

```diff
@@ -810,3 +811,32 @@ NULL
 
     return(res)
 }
+
+#' @rdname config_functions
+.config_collection_res <- function(source, collection, bands) {
```

Comment:
Consider use `.source_bands_resolution()` function

Reply:
Sure. I will replace it. Thanks!